spec/syntax.md

```diff
@@ -19,7 +19,8 @@
    1. [Variants](#variants)
    1. [Patterns](#patterns)
    1. [Expressions](#expressions)
-      1. [Reserved Sequences](#reserved)
+      1. [Private-Use Sequences](#private-use)
+      2. [Reserved Sequences](#reserved)
```

Comment:
We might want to start using explicit `<a name="foo"></a>` anchors at some point, rather than having two different names for spec sections.